beacon-chain/operations/attestations/service.go

```diff
@@ -26,11 +26,7 @@ type Config struct {
 // NewService instantiates a new attestation pool service instance that will
 // be registered into a running beacon node.
 func NewService(ctx context.Context, cfg *Config) (*Service, error) {
-	cache, err := ristretto.NewCache(&ristretto.Config{
-		NumCounters: forkChoiceProcessedRootsSize,
-		MaxCost:     forkChoiceProcessedRootsSize,
-		BufferItems: 64,
-	})
+	cache, err := lru.New(forkChoiceProcessedRootsSize)
```

Comment:
I think it's ok to bundle this change in the same PR, perhaps just mention it in the description?

Reply:
oh yes good idea